bw2parameters/ecoinvent_units.txt

```diff
@@ -0,0 +1,17 @@
+# default "ton" is interpreted as "short ton" (2000 pounds)
+ton = 1000 kg
```

Comment:
This is needed for ecoinvent but makes me nervous for general application...

Reply:
You mean because "short ton" is no longer defined? If you want, I can add it back under a different name